### ISSUE TITLE
jsf.formats is not a function

### ISSUE THREAD

**Issue by megamit** [json-schema-faker/json-schema-server#23]
Cannot provide any custom formats. Looks like a typo and meant to be `format` not `formats`

**Comment by pateketrueke** [comment]
You're talking about json-schema-faker (jsf) or json-schema-server?

**Comment by megamit** [comment]
My apologies this is a poorly written issue.
I have json-schema-server installed and running it with `--formats path` gives the error: 
```
Configuring json-schema-faker
TypeError: jsf.formats is not a function
```
I noticed in `server.js` you call `jsf.formats` (which doesnt exist) instead of `jsf.format`
see: https://github.com/json-schema-faker/json-schema-server/blob/master/lib/server.js#L110

**Comment by pateketrueke** [comment]
Got it, sorry, I need to update that package.

Can you wait for a new release?

**Comment by megamit** [comment]
Yup. For the time being i have a start script that sets up the server the same way as the cli and sets up the formats.

**Comment by pateketrueke** [comment]
Released patch as `0.4.2` 🎉 

Thanks for your feedback!
